src/patternfly/components/Tabs/tabs.scss

```diff
@@ -576,6 +623,7 @@ $pf-v6-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl
   text-decoration-line: none;
   background-color: var(--#{$tabs}__link--BackgroundColor);
   border-radius: var(--#{$tabs}__link--BorderRadius);
+  transition: background-color var(--#{$tabs}__link--TransitionDuration--background-color) var(--#{$tabs}__link--TransitionTimingFunction--background-color);
```

Comment:
Do you want to separate this rather than using the shorthand?

Reply:
Doh! Yep - fixed.


https://github.com/user-attachments/assets/a81d860e-2676-439c-89fe-566333edbdb1